zeitschrift-fur-medienwissenschaft.csl

```diff
@@ -1,16 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" class="note" default-locale="de-DE" demote-non-dropping-particle="sort-only">
-  <!--
-================================================================================
-   KOPFTEIL
-================================================================================
--->
   <info>
-    <title>Zeitschrift für Medienwissenschaft (German)</title>
-    <title-short>zfm</title-short>
```

Comment:
Okay, will put it back in.

> On 2015-01-04T14:23:07+0100, Philipp Zumstein said:
>    <info>

> -    <title>Zeitschrift für Medienwissenschaft (German)</title>
> -    <title-short>zfm</title-short>

> I wouldn't delete the short title.

> --- Reply to this email directly or view it on GitHub:
> https://github.com/citation-style-language/styles/pull/1313/files#r22439130

--
Till A. Heilmann, Dr. phil.

Online-Texte http://www.tillheilmann.info/german.php#OnlineTexts

Universität Siegen
Philosophische Fakultät
Medienwissenschaftliches Seminar
Adolf-Reichwein-Straße 2, Raum AR-K 611
57076 Siegen
Telefon: +49 271 740-4791
Telefax: +49 271 740-2731
www.uni-siegen.de/phil/medienwissenschaft/
www.multimediale-systeme.de